Doc/c-api/code.rst

```diff
@@ -76,3 +76,16 @@ bound into a function.
    information is not available for any particular element.
 
    Returns ``1`` if the function succeeds and 0 otherwise.
+
+.. c:function:: PyObject* PyCode_GetCode(PyObject *co)
```

Comment:
The two API giving code objects return type ``PyCodeObject*``:

* PyCode_NewEmpty()
* PyFrame_GetCode()

So it sounds more consistent to use ``PyCodeObject*`` for the parameter.